pkg/egressgateway/manager_privileged_test.go

```diff
@@ -237,6 +241,50 @@ func (k *EgressGatewayTestSuite) TestEgressGatewayManager(c *C) {
 		{ep1IP, destCIDR, egressCIDR1, testInterface1Idx},
 	})
 
+	// Add an additional Egress IP on the first interface, and a policy
+	// that selects this IP. The conflicting policy should be ignored,
+	// resulting in a "no gateway found" egress entry and no IP rule
+	// for ep3.
+	link, err := netlink.LinkByName(testInterface1)
+	if err != nil {
+		panic(err)
+	}
+
+	a, _ := netlink.ParseAddr(egressCIDR3)
+	netlink.AddrAdd(link, a)
+
+	addPolicy(c, k.policies, &policyParams{
```

Comment:
Nit: you can assign `policyParams` to `policy3 :=` or similar and cut down on the repetition to delete the policy.